test/650_proxy_env_test.sh

```diff
@@ -31,7 +31,7 @@ check
 # Check we can use weave env/config with unix -Hs specified
 weave_on $HOST1 stop
 weave_on $HOST1 launch-proxy -H unix:///var/run/weave/weave.sock
-assert_raises "run_on $HOST1 'eval \$(weave env) ; docker $CMD'"
-assert_raises "run_on $HOST1 'docker \$(weave config) $CMD'"
```

Comment:
But we should be connecting as `vagrant` user which is in the `docker` group, so it should be available, no?